pkg/config/branding/theme.go

```diff
@@ -196,7 +196,7 @@ func (c *ThemeClient) GetFile(ctx context.Context, themeID string, path string)
 	client := openapi.NewClientWithOptions(ctx, vc.Tenant, c.Client)
 
 	headers := openapi.Headers{Token: vc.Token}
```

Comment:
Initialize this as a pointer, so you can just pass the var in instead of doing &headers over and over again.